src/components/SignerPanel/SignerPanel.js

```diff
@@ -118,11 +166,9 @@ class SignerPanel extends React.Component {
         await this.signTransaction(pretransaction, intent)
       }
 
-      const transactionRes = await this.signTransaction(transaction, intent)
-      // Create new notification
-      onTransactionSuccess && onTransactionSuccess(transaction)
+      const transactionHash = await this.signTransaction(transaction, intent)
 
-      transactionBag.accept(transactionRes)
+      transactionBag.accept(transactionHash, intent)
```

Comment:
Not sure if it was intended to pass back `intent`, but this isn't used by `@aragon/wrapper`.